modules/core/src/alloc.cpp

```diff
@@ -130,39 +129,14 @@ void* fastMalloc_(size_t size)
 void* fastMalloc(size_t size)
 #endif
 {
-#ifdef HAVE_POSIX_MEMALIGN
-    if (isAlignedAllocationEnabled())
-    {
-        void* ptr = NULL;
-        if(posix_memalign(&ptr, CV_MALLOC_ALIGN, size))
-            ptr = NULL;
-        if(!ptr)
-            return OutOfMemoryError(size);
-        return ptr;
-    }
-#elif defined HAVE_MEMALIGN
-    if (isAlignedAllocationEnabled())
-    {
-        void* ptr = memalign(CV_MALLOC_ALIGN, size);
-        if(!ptr)
-            return OutOfMemoryError(size);
-        return ptr;
-    }
-#elif defined HAVE_WIN32_ALIGNED_MALLOC
-    if (isAlignedAllocationEnabled())
```

Comment:
I wornder if runtime / build time switcher for disabling aligned allocation are still relevant